app/config/scopes.php

```diff
@@ -1,6 +1,12 @@
 <?php
 
 return [ // List of publicly visible scopes
+    'account' => [
+        'description' => 'Access to make actions on behalf of a user account',
+    ],
+    'sessions' => [
```

Comment:
similarly why is this not `sessions.write` ?

Reply:
We can change this.

Reply:
Changed